pkgs/os-specific/linux/firmware/raspberrypi-wireless/default.nix

```
@@ -32,20 +32,19 @@ stdenv.mkDerivation {
     mkdir -p "$out/lib/firmware/brcm"
 
     # Wifi firmware
-    shopt -s extglob
-    for filename in firmware-nonfree/brcm/brcmfmac434??{,s}-sdio.*; do
-      cp "$filename" "$out/lib/firmware/brcm"
-    done
+    cp -rv /build/firmware-nonfree/debian/config/brcm80211/. "$out/lib/firmware/"
```

```suggestion
    cp -rv $NIX_BUILD_TOP/firmware-nonfree/debian/config/brcm80211/. "$out/lib/firmware/"
```

While in practice it ends-up being `/build`, I believe it shouldn't be relied upon. I think this could break some usage types, like `nix-shell` or `nix develop`, too.